modules/RsIskraMeter/src/main.rs

```diff
@@ -893,6 +1041,223 @@ impl generated::OnReadySubscriber for IskraMeter {
 
 impl generated::SerialCommunicationHubClientSubscriber for IskraMeter {}
 
+impl generated::DisplayMessageServiceSubscriber for IskraMeter {
+    fn clear_display_message(
+        &self,
+        _context: &generated::Context,
+        _request: generated::types::display_message::ClearDisplayMessageRequest,
```

Comment:
can you include the generated::types::display_message at the beginning of the file?